### PR TITLE
naive forcasting docstring edits

### PR DESCRIPTION
**Pull request by AreloTanoh**
#### Reference Issues/PRs

#1245 

#### What does this implement/fix? Explain your changes.

Removed  "option" from argument parentheses.
Made other minor edits to conform to docstring standards.
Applied to naive.py (18/08/21) and afterwards trend.py. (19/08/21)

##### For all contributions
- [x] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [ ] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1162,6 +1162,15 @@
       "contributions": [
         "doc"
       ]
+    },
+       {
+      "login": "AreloTanoh",
+      "name": "Arelo Tanoh",
+      "avatar_url": " ",
```
Comment (mloning) [inline .all-contributorsrc:1169]: that's the link to your user picture from your account 
```suggestion
      "avatar_url": "https://avatars.githubusercontent.com/u/87912196?v=4",
```

**Review by mloning — COMMENTED** [review]
Looks all good @AreloTanoh, just left one minor comment

File: .all-contributorsrc (hunk 1 above)
Comment (AreloTanoh) [reply .all-contributorsrc:1169]: Ah i see. How do i leave that blank ?

Comment (AreloTanoh) [reply .all-contributorsrc]: @mloning made some changes so hope will work

**Review by fkiraly — APPROVED** [review]
looks good to me, thanks!

**Pull request merged**